components/badge/badge.css

```diff
@@ -0,0 +1,31 @@
+.online {
+  background-color: #00ff00;
+}
+.offline {
+  background-color: #ff0000;
+}
+.idle {
+  background-color: #868686;
+}
+.av_badge_lg,
+.av_badge_md,
+.av_badge_sm {
+  position: absolute;
+  /* border-radius: 50%; */
```

Comment:
remove the commented code

Reply:
Thanks for the review, will resolve the changes